x-pack/platform/plugins/shared/actions/server/lib/action_executor.ts

```diff
@@ -418,7 +418,7 @@ export class ActionExecutor {
         const actionType = actionTypeRegistry.get(actionTypeId);
         const configurationUtilities = actionTypeRegistry.getUtils();
 
-        let validatedParams;
+        let validatedParams: Record<string, unknown>;
```

Comment:
Odd that this typing was needed here, but not on the config or secrets, without some code change that would make it neccessary.  Can it be removed to simplify the PR?

Reply:
Just leaving it here as we discussed it on slack, this specifically is always casted to this specific type in functions that comes later in the codebase, since I needed to access it I did not see any need to not simply add the correct type from the start.